<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportHistory</class>
 <widget class="QWidget" name="ImportHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="oldGG">
      <attribute name="title">
       <string>Old GG</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Gadu Gadu 7.0 and olders:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hint_old">
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Path to gadu-gadu archives file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ggPath">
           <property name="toolTip">
            <string>Exact path to archives.dat file.
File can be found in Gadu-Gadu's profile directory.</string>
           </property>
           <property name="whatsThis">
            <string>Exact path to archives.dat file.
File can be found in Gadu-Gadu's profile directory.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ggBrowseButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="newGG">
      <attribute name="title">
       <string>New GG</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gadu Gadu version 8, 9 and 10:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hint_new">
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Path to gadu-gadu profile dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="gg8Path">
           <property name="toolTip">
            <string>Path to Gadu-Gadu user's profile directory.
Given directory should contain Archive.db file and ImgCache subdirectory.</string>
           </property>
           <property name="whatsThis">
            <string>Path to Gadu-Gadu user's profile directory.
Given directory should contain Archive.db file and ImgCache subdirectory.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gg8BrowseButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="5">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Target account:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="proceedButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="accountCBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
